woocommerce/woocommerce
# [WIP] Update pnpm to latest and update composer deps to avoid postinstall error

### Changes proposed in this Pull Request:

* Updated pnpm to 8.6.0, which has a new lock file version
* Regenerated lockfile for new version
* Update ref to pnpm version in the monorepo setup CI action

I'm making this change now because already pnpm is fast getting out of date. I encountered issues when swapping between version `8.3.1` which we use in CI and `8.6.0` due to changes in the lockfile (global install failed due to some issue with the lockfile when generated). Since devs can easily find themselves on 8.6.0 and get weird errors I figured we should just move to latest version now.

### How to test the changes in this Pull Request:

* All CI checks passing should be adequate

Note: We will probably want to send out some kind of minor blog post / woo dev blog announcement / p2 to let devs know they will need to update again.
